lib/puppet/util/autoload.rb

```diff
@@ -168,8 +156,6 @@ def search_directories(env)
       # --cprice 2012-03-16
       if Puppet.settings.app_defaults_initialized?
         unless @initialized
```

Comment:
I don't think this block is doing anything now?

Reply:
this is what happens when i'm juggling too many buffers